Server/Controllers/MatchController.cs

```diff
@@ -258,7 +258,27 @@ public async Task<ActionResult> After([FromBody] AfterMatchRequest request)
 
             return Ok();
         }
-
+        [HttpPost]
+        [Route("patreon_player_update")]
```

Comment:
Use kebab-case for routes
```suggestion
        [Route("patreon-player-update")]
```